src/database/mod.rs

```diff
@@ -158,6 +168,73 @@ pub trait Database: BatchOperations {
     ///
     /// It should insert and return `0` if not present in the database
     fn increment_last_index(&mut self, keychain: KeychainKind) -> Result<u32, Error>;
+
+    /// Delete UTXOs provided as params, if `spent_utxos` is empty it deletes all spent
+    /// UTXOs in the database.
```

Comment:
Something like this can sound clearer?
```suggestion
    /// Delete a list of spent utxos from database. Delete all spent utxos if  list is `None`. 
```

Reply:
Sure, will make the change

Reply:
Sure, will make the change